### PR TITLE
JDP200801-17

### PR DESCRIPTION
**Pull request by Greem666**
Includes changes introduced in JDP200801-28, and indirectly JDP200801-29 (due to problems with Group Entity implementation). Also required slight alterations due to conflicts with Cart-User relationship, which was implemented after branch -17 was created.

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/domain/group/GroupEntity.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity
+@Table(name="PRODUCT_GROUPS")
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class GroupEntity {
```
Comment (piotr-grosicki) [inline src/main/java/com/kodilla/ecommercee/domain/group/GroupEntity.java:18]: to zadanie nie powinno zawierac zmian w tym pliku

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]
konflikty

**Review by piotr-grosicki — CHANGES_REQUESTED** [review]
konflikt ;)

File: src/main/java/com/kodilla/ecommercee/domain/group/GroupEntity.java (hunk 1 above)
Comment (Greem666) [reply src/main/java/com/kodilla/ecommercee/domain/group/GroupEntity.java]: Poprawione, zmiany przeniesiono do JDP200801-29.

**Comment by Greem666** [comment]
> konflikt ;)

Poprawione po pull`u z mastera.

**Comment by Greem666** [comment]
> > konflikt ;)
> 
> Poprawione po pull`u z mastera.

**Review by piotr-grosicki — APPROVED** [review]

**Pull request merged**